### ISSUE TITLE
Concat unable to quantized in onnxruntime?

### ISSUE THREAD

**Issue by jinfagang** [microsoft/onnxruntime#11242]
I met all concat scene and it generates something ike this after quantize:

<img width="794" alt="image" src="https://user-images.githubusercontent.com/21303438/163718614-0baded99-5d63-4c4e-a149-c162d89e7363.png">

```
onnxruntime/capi/onnxruntime_inference_collection.py", line 381, in _create_inference_session
    sess.initialize_session(providers, provider_options, disabled_optimizers)
onnxruntime.capi.onnxruntime_pybind11_state.RuntimeException: [ONNXRuntimeError] : 6 : RUNTIME_EXCEPTION : Exception during initialization: /Users/jintian/dev/libs/onnxruntime/onnxruntime/contrib_ops/cpu/quantization/qlinear_concat.cc:36 onnxruntime::contrib::QLinearConcat::QLinearConcat(const onnxruntime::OpKernelInfo &) input_def_count >= 8 && (input_def_count - 2) % 3 == 0 was false. At least two inputs are needed, and each input must be (tensor, scale, zero_point) tuple!
```

All concat like this, can not be inference at all..... How to resolve it?

**Comment by yufenglee** [comment]
Concat can be quantized. Could you please provide a model to repro your issue?

**Comment by yufenglee** [comment]
Does Concat in your model only have 1 input? QLinearConcat has a check that requires at least 2 input. We will change it to allow 1 input. However, why do you need a Concat with only 1 input?

**Comment by jinfagang** [comment]
@yufenglee  thanks for your notice. Let me given more context, I have a sort of complicated model produce `masks`, `scores`, `labels` which is doing instance segmentation, I exported to onnx, and onnx forward correctly, then I want using out to quantize it.

Current situation is, using ort quatize API, I get the model, which the last concat -> QConcat but with only on input.

And the model is unable to inference at all obviously.

So:


1. I now the node is 1 input, but I can not control it, it was generated by ort API;
2. the original concat in flaot32, is just single input since it was doing something like `torch.stack`, and this is again, not controlled by me, it was exported by pytorch.

Anyway to resolve it?

**Comment by jinfagang** [comment]
@yufenglee  thanks for your notice. Let me given more context, I have a sort of complicated model produce `masks`, `scores`, `labels` which is doing instance segmentation, I exported to onnx, and onnx forward correctly, then I want using out to quantize it.

Current situation is, using ort quatize API, I get the model, which the last concat -> QConcat but with only on input.

And the model is unable to inference at all obviously.

So:


1. I now the node is 1 input, but I can not control it, it was generated by ort API;
2. the original concat in flaot32, is just single input since it was doing something like `torch.stack`, and this is again, not controlled by me, it was exported by pytorch.

Anyway to resolve it?

**Comment by yufenglee** [comment]
PR#11265 to fix it.

**Comment by jinfagang** [comment]
@yufenglee thanks. when will the PR be landed?

**Comment by yufenglee** [comment]
PR was merged.
